### PR TITLE
[DRAFT] Throws Garou a bone & adjust some spells

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request

After witnessing a complete and utter dumpster fire of a round, and numerous Garou acting smug and valiant that nothing can be done about them, something is getting done about them. This PR adjusts the healing costs and gnosis costs for a number of gifts to make them less spamable in combat.

also, some facts:

            if("Homid")
                character.auspice.gnosis = 1
                character.auspice.start_gnosis = 1
                character.auspice.base_breed = "Homid"
            if("Lupus")
                character.auspice.gnosis = 5
                character.auspice.start_gnosis = 5
                character.auspice.base_breed = "Lupus"
            if("Metis")
                character.auspice.gnosis = 3
                character.auspice.start_gnosis = 3
                character.auspice.base_breed = "Crinos"

Generally, you are going to hover around 3 gnosis in warform, this makes gifts more impactful

## Why It's Good For The Game

To put it simply; Garou have provided a rather unpleasent environment on the server, due to how their crit is broken and their mechanics, they effectively get an aheal. This isn't good.

## Changelog
:cl:
Tweaks the power of some silver bullets, allowing them to have a higher probability of stunning and lowering gnosis.
Ups the stun time of silver bullets. does not increase damage.
Reimplements a cost for using the stoic pose, making it more of a get out of jail free card but gives them more time to be in stasis, you need to make it count.
Bloody feast and freezing wind get more of a gnosis cost.
Claws get a damage up in turn.
Longer up time cost on the aoe stun for glassies.

Compensates with the Set mummy form also getting a nerf too. Sorry setites but it had to be done.
/:cl:

### CODE REVIEW

**Comment by AllCorruptingWyrm** [comment]
also nearly forgot; Rage heal restores more HP but has a MUCH higher cost, spamming it will eventually attrition you out.

**Comment by CatandWet** [comment]
Buffs a broken skill, does a bunch of things that won't actually solve garou balance; it helps to know how things work before you change them.
Slightly higher rage costs are a good idea but the gnosis changes do not work; all homid born garou basically cannot use 80% of tribe gifts with these changes.

**Comment by AllCorruptingWyrm** [comment]
"It helps to know things before you change them" really is not anything a staff member should be saying to contributors who are trying to make this place better, I know you had your PR so do it but dont get nasty with me because its a collaborative team effort.

**Comment by CatandWet** [comment]
It was a little snappy, my bad. I do really appreciate you trying to get them toned down and I am happy to use your PR's changes and ideas because a fair bit of it is good and well founded.
Gnosis almost needs an entire rework before it can really be used as a resource for gifts.

File: code/modules/vtmb/werewolf/tribes.dm [hunk 1]
```diff
@@ -11,31 +12,31 @@
 		var/mob/living/carbon/C = owner
 		if(isgarou(C))
 			var/obj/were_ice/W = new (get_turf(owner))
-			C.Stun(200)
+			C.Stun(400)
 			C.forceMove(W)
-			spawn(200)
+			spawn(400)
 				C.forceMove(get_turf(W))
 				qdel(W)
```
Comment (Metekillot) [inline code/modules/vtmb/werewolf/tribes.dm:19]: move this into a proc that gets called with our timers

File: code/modules/vtmb/werewolf/tribes.dm [hunk 2]
```diff
@@ -11,31 +12,31 @@
...
 		if(islupus(C))
 			var/obj/were_ice/lupus/W = new (get_turf(owner))
-			C.Stun(200)
+			C.Stun(400)
 			C.forceMove(W)
-			spawn(200)
+			spawn(400)
 				C.forceMove(get_turf(W))
 				qdel(W)
```
Comment (Metekillot) [inline code/modules/vtmb/werewolf/tribes.dm:33]: Delete all of this and just make the initial check a || for these two conditions

File: code/modules/wod13/ammostack.dm [hunk 3]
```diff
@@ -328,10 +328,10 @@
 	if(iswerewolf(target) || isgarou(target))
 		var/mob/living/carbon/M = target
 		if(M.auspice.gnosis)
-			if(prob(40))
+			if(prob(75))
 				adjust_gnosis(-1, M)
 		else
-			M.Stun(10)
+			M.Stun(20)
```
Comment (Metekillot) [inline code/modules/wod13/ammostack.dm:334]: hardstuns are a no from me, please just delete this bit for hardstunning someone with special ammo, stuns are lethal

File: code/modules/wod13/ammostack.dm [hunk 4]
```diff
@@ -343,10 +343,10 @@
...
```
Comment (Metekillot) [inline code/modules/wod13/ammostack.dm:349]: see above

File: code/modules/wod13/ammostack.dm [hunk 5]
```diff
@@ -358,10 +358,10 @@
...
```
Comment (Metekillot) [inline code/modules/wod13/ammostack.dm:364]: three! three hard stuns! ah ah ah! please remove

File: code/modules/wod13/ammostack.dm [hunk 6]
```diff
@@ -375,10 +375,10 @@
...
```
Comment (Metekillot) [inline code/modules/wod13/ammostack.dm:381]: please remove this

**Review by Metekillot — CHANGES_REQUESTED** [review]

**Comment by XanderDox** [comment]
Are you still working on this @AllCorruptingWyrm ?

**Comment by XeonMations** [comment]
Closed due to the author not having time.
